Tools/LibEnsemble/sim/inputs

```diff
@@ -195,25 +195,25 @@ warpx.mirror_z_npoints = 4 4
 #################################
 ############# LASER #############
 #################################
-lasers.nlasers     = 2
-lasers.names        = laser1 laser2
+lasers.nlasers = 2
```

Comment:
this file contains no changes but white spaces. do we want to change it or reset the changes in it?

Reply:
Hi @ax3l !
These changes were triggered by:
https://github.com/ECP-WarpX/WarpX/pull/1141#discussion_r449362150.
The way we then replace this parameters in LibEnsemble depends on the spaces, that is why it is important to correct them.
So we want to keep the spaces changes.